ospd_openvas/daemon.py

```diff
@@ -211,10 +219,136 @@
+        'name': 'max_hosts',
+        'default': 30,
+        'mandatory': 0,
+        'visible_for_client': 0,
+        'description': (
+            'Is maximum number of hosts to test at the same time which'
+            + 'should be given to the client (which can override it).'
+            + 'This value must be computed given  your bandwidth,'
+            + 'the number of hosts you want to test, your amount of'
+            + 'memory and the horsepower of your processor(s).'
+        ),
+    },
+    'max_checks': {
+        'type': 'integer',
+        'name': 'max_checks',
+        'default': 10,
+        'mandatory': 0,
+        'visible_for_client': 0,
+        'description': (
+            'is the number of plugins that will run against each host being'
+            + ' tested. Note that the total number of process will be max'
+            + ' checks x max_hosts so you need to find a balance between'
+            + ' these two options. Note that launching too many plugins at'
+            + ' the same time may disable the remote host, either temporarily'
+            + ' (ie: inetd closes its ports) or definitely (the remote host'
+            + ' crash because it is asked to do too many things at the'
+            + ' same time), so be careful.'
+        ),
+    },
+    'port_range': {
+        'type': 'string',
+        'name': 'port_range',
+        'default': '',
+        'mandatory': 0,
+        'visible_for_client': 0,
+        'description': (
+            'This is the default range of ports that the scanner plugins will '
+            + 'probe. The syntax of this option is flexible, it can be a '
+            + 'single range ("1-1500"), several ports ("21,23,80"), several '
+            + 'ranges of ports ("1-1500,32000-33000"). Note that you can '
+            + 'specify UDP and TCP ports by prefixing each range by T or U. '
+            + 'For instance, the following range will make openvas scan UDP '
+            + ' ports 1 to 1024 and TCP ports 1 to 65535 : '
+            + '"T:1-65535,U:1-1024".'
+        ),
+    },
+    'test_alive_hosts_only': {
+        'type': 'boolean',
+        'name': 'test_alive_hosts_only',
+        'default': 0,
+        'mandatory': 0,
+        'visible_for_client': 0,
+        'description': (
+            'If this option is set, openvas will scan the target list for '
+            + 'alive hosts in a separate process while only testing those '
+            + 'hosts which are identified as alive. This boosts the scan '
+            + 'speed of target ranges with a high amount of dead hosts '
+            + 'significantly.'
+        ),
+    },
+    'source_iface': {
+        'type': 'string',
+        'name': 'source_iface',
+        'default': '',
+        'mandatory': 0,
+        'visible_for_client': 0,
+        'description': (
+            'Name of the network interface that will be used as the source '
+            + 'of connections established by OpenVAS. The scan won\'t be '
```

Comment:
```suggestion
            + 'of connections established by openvas. The scan won\'t be '
```

As an alternative update the other occurrences of `openvas` to use `OpenVAS`